pkgs/by-name/pl/plymouth-matrix-theme/package.nix

```
@@ -0,0 +1,46 @@
+{
+  stdenvNoCC,
+  fetchFromGitHub,
+  lib,
+  unstableGitUpdater,
+}:
+stdenvNoCC.mkDerivation {
+  pname = "plymouth-matrix-theme";
+  version = "0.1.0-unstable-19-02-2017";
+  src = fetchFromGitHub {
+    owner = "storax";
+    repo = "plymouth-matrix-theme";
+    rev = "b2268f25dea7537ed5709b00d5a83b3600265c54";
+    hash = "sha256-JmMmpw1By5U6OTaSPnJOZZxrieSnXivMmdt/JPazjpI=";
+  };
+
+  dontBuild = true;
+
+  postPatch = ''
+    # Remove not needed files
+    rm README.rst LICENSE Makefile
+  '';
```

```suggestion
  postPatch = ''
    # Remove not needed files
    rm README.rst LICENSE Makefile
  '';

  dontBuild = true;
```
Re-order by phases order (patch patch preceding configure/build phase.)